docs/components/table/README.md

```diff
@@ -382,12 +383,28 @@ will emit the `filtered` event, passing a single argument which is the complete
 items passing the filter routine. Treat this argument as read-only.
 
 ### Sorting
+As mentioned above in the **fields** section, you can make columns sortable. Clciking on
+sortable a column header will sort the column in ascending direction, while clicking
+on it again will switch the direction or sorting.  Clicking on a non-sortable column
```

Comment:
Alo, I could dd a button that says 'clear sort' (which is disabled if `sort-by` is `null`) to the example.

Clicking on the button would set `sort-by` to `null`, and if `sort-by` is `null` then the button could be disabled.  Sowing a bit more interactivity with with the table.